server.py

```
@@ -19,13 +28,15 @@
 from sqlalchemy import or_
 
 # --- Local Application Imports --- #
-from db_models import db, Admins, Articles
+from db_models import db, Admins, Articles, Products
 from utils import *
```

Ruff (0.12.2)

`from utils import *` used; unable to detect undefined names

(F403)